### PR TITLE
Messege box

### PR DESCRIPTION
**Pull request by GoralTomaszGorski**
message from prepared typos to fix,
controller is ready 
but still problem with MessageService tomorrow I will work on it

### CODE REVIEW

**Comment by GoralTomaszGorski** [comment]
[aea1ed3](https://github.com/GoralTomaszGorski/psychotherapist_office/pull/21/commits/aea1ed3ef785bfe01554aa04bdfb7ed7c28f92c3)
to repair tommorow

**Comment by GoralTomaszGorski** [comment]
however, today I made this box

fuc***g token from gmail was necessary ;)

File: src/main/java/goral/psychotherapistoffice/domain/messeges/MessageService.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+        //logic
+        //smtp properties
+        Properties properties = new Properties();
+        properties.put("mail.smtp.auth", true);
+        properties.put("mail.smtp.starttls.enable", true);
+        properties.put("mail.smtp.port", "587");
+        properties.put("mail.smtp.host", "smtp.gmail.com");
+
+        final String username = "tomasz.gorski88@gmail.com";
```
Comment (gorskimarcin96) [inline src/main/java/goral/psychotherapistoffice/domain/messeges/MessageService.java]: in php usually we are creating hide .env file and we can move this values to file

https://www.geeksforgeeks.org/environment-variables-in-java/

**Review by gorskimarcin96 — APPROVED** [review]

**Comment by GoralTomaszGorski** [comment]
the methods have been split into another class to clean up the code with token and email

**Pull request merged**